pyproject.toml

```diff
@@ -1,7 +1,11 @@
 [tool.black]
 line-length = 100
 
+[tool.isort]
+profile = "black"
+
 [tool.mypy]
+python_version = "3.10"
```

Comment:
This is probably too late in the game to care, but just noticed here. Does using 3.10 cause issues for people without that Python version? I know some of our tests still expect / use Python 3.6.